packages/react-core/src/components/Nav/NavExpandable.tsx

```diff
@@ -110,6 +125,7 @@ export class NavExpandable extends React.Component<NavExpandableProps, NavExpand
               isActive && styles.modifiers.current,
               className
             )}
+            {...getOUIAProps('NavItem', ouiaId, ouiaSafe)}
```

Comment:
I couldn't pass tests if I put `useOUIAProps`. I suppose it should be done in the same as here https://github.com/patternfly/patternfly-react/blob/master/packages/react-core/src/components/Nav/Nav.tsx#L71

Reply:
Yep! Please use `useOUIAProps` instead since that has bugfixes!